lib/simple_lru.js

```diff
@@ -43,10 +44,12 @@ Cache.prototype.get = function(key,hit){
      * it to the head of linked list  
      */
     hit = hit != undefined && hit != null ? hit : true;
-    if(cacheVal && hit)
+
+    if(cacheVal && hit && (!cacheVal.time || (Date.now() - cacheVal.time) < this.maxTime)) 
```

Comment:
But we need to test to update the cacheValue, think there is a bug there. Not setting a new timestamp :-)